infoschema/tables.go

```diff
@@ -49,8 +49,9 @@ const (
 	// TableSchemata is the string constant of infoschema table.
 	TableSchemata = "SCHEMATA"
 	// TableTables is the string constant of infoschema table.
-	TableTables           = "TABLES"
-	tableColumns          = "COLUMNS"
+	TableTables = "TABLES"
+	// TableColumns is the string constant of infoschema table
```

Comment:
This comment seems redundant？

Reply:
it seems public constant before and after this line has comment like this, so add this comment, need i remove it?